WordPress/gutenberg
# Use android artifacts for react-native libraries in react-native-bridge

## What?


This PR updates the `react-native-editor`, `react-native-aztec` & `react-native-bridge` projects to use the published artifacts of `react-native` & `react-native` libraries such as `react-native-svg`.

## Why?


This is done to simplify these 3 projects and more importantly so that we don't have to maintain forks for them. This change will be followed up with https://github.com/WordPress/gutenberg/pull/39348 which will drop these forks and use the original repos instead.

## How?


We already had a setup for using these published artifacts in these projects. This was used when we were publishing `react-native-bridge` & `react-native-aztec` projects themselves. So, we just needed to remove the setup for using the source code for these libraries.

## Testing Instructions





We should verify that both `react-native-editor` demo project and WordPress-Android work as expected. There isn't a specific change in this PR, so we'll need to make due with smoke tests.
